openprescribing/frontend/price_per_unit/profile.py

```diff
@@ -0,0 +1,37 @@
+"""
+Basic profiling code for working out where we're spending our time
+
+Invoke with:
+./manage.py shell -c 'from frontend.price_per_unit.profile import profile; profile()'
+"""
+from cProfile import Profile
+import datetime
+import time
+
+from .savings import get_all_savings_for_orgs
+
+
+def test():
+    get_all_savings_for_orgs("2019-11-01", "ccg", ["99C"])
+    # get_all_savings_for_orgs("2019-11-01", "all_standard_practices", [None])
+
+
+def profile():
+    num_attempts = 5
+    attempts = []
+    for _ in range(num_attempts):
+        profiler = Profile()
+        start = time.time()
+        profiler.runcall(test)
+        duration = time.time() - start
+        attempts.append((duration, profiler))
+    attempts.sort()
+    profile_file = "profile.{}.prof".format(
+        datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+    )
+    attempts[0][1].dump_stats(profile_file)
+    print(
+        "{}s (best of {}), profile saved as: {}".format(
+            attempts[0][0], num_attempts, profile_file
+        )
+    )
```

Comment:
I think we can probably drop this? It's neat but I bet we never use it again

Reply:
Actually, I've had a similar thing around for a while as I've been working on the MatrixStore and I've ended up using it so often I thought I ought to commit it!

Reply:
OK - in that case, what about mentioning this somewhere - perhaps the MatrixStore README, as an example of profiling? I can imagine it'll easily get lost otherwise.